src/packages/user/user/collection/user-collection.element.ts

```diff
@@ -1,59 +1,24 @@
 import { UmbUserCollectionContext } from './user-collection.context.js';
-import { css, html, customElement, state } from '@umbraco-cms/backoffice/external/lit';
-import { UmbTextStyles } from "@umbraco-cms/backoffice/style";
-import { UMB_COLLECTION_CONTEXT } from '@umbraco-cms/backoffice/collection';
-import { UmbLitElement } from '@umbraco-cms/internal/lit-element';
-import type { UmbRoute } from '@umbraco-cms/backoffice/router';
+import { html, customElement } from '@umbraco-cms/backoffice/external/lit';
+import { UmbTextStyles } from '@umbraco-cms/backoffice/style';
+import { UMB_COLLECTION_CONTEXT, UmbCollectionElement } from '@umbraco-cms/backoffice/collection';
 
 import './user-collection-header.element.js';
 
-export type UsersViewType = 'list' | 'grid';
 @customElement('umb-user-collection')
-export class UmbUserCollectionElement extends UmbLitElement {
-	#collectionContext = new UmbUserCollectionContext(this);
+export class UmbUserCollectionElement extends UmbCollectionElement {
+	public collectionContext = new UmbUserCollectionContext(this);
 
-	@state()
-	private _routes: UmbRoute[] = [
-		{
-			path: 'grid',
-			component: () => import('./views/grid/user-collection-grid-view.element.js'),
-		},
-		{
-			path: 'list',
-			component: () => import('./views/table/user-collection-table-view.element.js'),
-		},
-		{
-			path: '',
-			redirectTo: 'grid',
-		},
-	];
-
-	connectedCallback(): void {
-		super.connectedCallback();
-		this.provideContext(UMB_COLLECTION_CONTEXT, this.#collectionContext);
+	constructor() {
+		super();
+		this.provideContext(UMB_COLLECTION_CONTEXT, this.collectionContext);
```

Comment:
As well as making the CollectionContext a Controller then it should also become self providing :-) so removing this line.

Reply:
We also need to figure out if it makes sense to have a generic element for Collections, cause we have "workspace-view-collection.element" but maybe it not good to have one generic.. and BTW. If someone needs their own custom, then it might be user collection.. so I don't know, just a perspective.

Reply:
Oh, I see you did that with the umb-collection element, right?

Reply:
There is one generic `UmbCollectionElement` and the user collections extend that. I then overwrite the renderToolbar method to add the custom filters. I am trying to move as much back into the generic element that makes sense. 

I think we should have an extension something called a "CollectionAction". In most cases, this will be the create action, but for users, it will be both create and invite.

I will also add logic for pagination to the generic context/element.

Reply:
I have made the collection context self providing. Thanks 👍